events/managers.py

```diff
@@ -132,7 +132,7 @@ def buy_a_bet(self, user, event_id, for_outcome, price):
 
         user.total_cash -= bought_for_total
         user.portfolio_value += bought_for_total
-        user.save(update_fields=['total_cash'])
+        user.save()
```

Comment:
Nie mam pojęcia co to jest, ale widzę że tego jest więcej. Trzeba to sprawdzić!!!